homeassistant/components/homeassistant/triggers/homeassistant.py

```diff
@@ -3,7 +3,7 @@
 import voluptuous as vol
 
 from homeassistant.const import CONF_EVENT, CONF_PLATFORM
-from homeassistant.core import CALLBACK_TYPE, HassJob, HomeAssistant
+from homeassistant.core import CALLBACK_TYPE, DOMAIN, HassJob, HomeAssistant
```

Comment:
I think we this integration's DOMAIN constant
```suggestion
from homeassistant.core import CALLBACK_TYPE, HassJob, HomeAssistant
...
from .const import DOMAIN
```

Reply:
In fact we can import from `..const`